### DISCUSSION TITLE
More sorting modes

### DISCUSSION THREAD

**Discussion by aha999** [signalstickers/signalstickers#844] [Ideas / feature request]
Please add more sorting modes, with up/down buttons to change order like:

- Most popular/least popular (when users press "get pack" you could count that as a download, therefore being more popular)
- by oldest/newest
- Alphabetically

etc...

**Comment by darkobits** [comment]
Thanks for the input, mate!

We can definitely consider adding a reversible alpha-sort and newness sort for sticker packs. 👍 

However, we don't track things like downloads (in fact, we collect almost no data about our users or their behavior on the site) so implementing a popularity sort would not be feasible at this time.

**Comment by romainricard** [comment]
Thanks for the input :+1: 

You can find what we collect [here](https://signalstickers.com/about#privacy-policy). Signalstickers's stats are [publicly available](https://ping.signalstickers.com/), but this system is not reliable enough to be able to propose a popularity sorting yet. Furthermore, there are some technical issues related to the way we host our website.

**Comment by bentolor** [comment]
I'd miss **filtering per sticker count in a pack**: 

I'd love to get rid of all those low quality packs like https://signalstickers.com/pack/bf93cf2841dd915a01435936ea226dd7

Even better – and I think that's the intent of the original author – would be ways to sort/filter on quality/feedback. But I currently have no idea how to implement that in a simple ways. I'd assume this would require some analytics.

**Comment by Meteor0id** [comment]
If we sort by popular, I recommend to make it most popular in the last 3 or 6 months rather than most popular all time, otherwise older sticker packs will always be at the top.
